lib/core/osal/netdev_funcs.c

```diff
@@ -236,11 +238,41 @@ netdev_get_offloads(const char *ifname, struct offloads *off)
     return ret;
 }
 
+static void
+netdev_get_channels_from_sysfs(const char *if_name, uint32_t *rxq, uint32_t *txq)
+{
+    char buf[PATH_MAX];
+    struct dirent *entry;
+    DIR *dir;
+    int ret;
+
+    *rxq = *txq = 0;
+
+    ret = snprintf(buf, PATH_MAX, "/sys/class/net/%s/queues/", if_name);
+    if (ret)
+        return;
+
+    dir = opendir(buf);
+    if (!dir)
+        return;
```

Comment:
Not sure this return needs to be converted to CNE_RET() macro, but maybe it will just print out a message we do not need to see.

Reply:
I'll update it

Reply:
cne_log depends on osal - so I can move things around to be able to do this - but it will involve some rejig of some files

Reply:
leave it for now